src/recursion.cpp

```diff
@@ -749,9 +749,11 @@ double class_tree::prior_transition(int s, int t, int level)
     if( s == 1 )   // alternative
     {
         if( t == 1 )  // alternative
-            return( log( 1.0 - eta) + log(gamma) -level*log(beta)  );    
+            // return( log( 1.0 - eta) + log(gamma) -level*log(beta)  );
+            return( log(1.0-eta) + log(beta)  ); 
```

Comment:
I think that the transition matrix should have the structure of the MRS paper as a special case. The proposed matrix does not have that structure.